src/app/portlets/dot-edit-content/dot-edit-content.component.ts

```diff
@@ -191,7 +191,9 @@ export class DotEditContentComponent implements OnInit {
     private removeContentlet($event: any): void {
         this.dotConfirmationService.confirm({
             accept: () => {
-                this.dotEditContentHtmlService.removeContentlet($event.dataset.dotInode);
+                this.dotEditContentHtmlService.removeContentlet($event.dataset.dotContainerIdentifier,
```

Comment:
This should receive 2 params, container and contentlet, both an object

We already have a interface: ```DotPageContainer``` we need the same for the contentlets.